lib/poller.py

```diff
@@ -26,14 +27,18 @@ def poll(self):
 
         entries = self.sierra_client.holdshelf_entries()
 
-        self.logger.info(f'Found {len(entries)} holdshelf entries')
+        unfiltered_entries_count = len(entries)
         try:
             entries = list(filter(self.unprocessed, entries))
         except Exception as error:
             self.logger.error(f'Redis read error; Aborting. Error: {error}')
             return
+        self.logger.info(
+            f'Found {unfiltered_entries_count} holdshelf entries' +
+            f' - {len(entries)} of them new')
 
-        self.send_notifications(entries)
+        if os.environ.get('DISABLE_NOTIFICATIONS', 'false') == 'false':
```

Comment:
is there a reason to use `== 'false'` as the default instead of `is False`?

Reply:
`os.environ.get` always returns strings, so if `DISABLE_NOTIFICATIONS=false`, then `os.environ.get('DISABLE_NOTIFICATIONS') == 'false'`